### PR TITLE
Translated pt locations files

### PR DESCRIPTION
**Pull request by douglara**

### CODE REVIEW

File: app/locales/pt/locationTracking.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    "logging_message":"It is currently logging your location privately every five minutes. Your location information will NOT leave your phone.",
-    "not_logging_message":"NOTE: After clicking this button you may be prompted to grant Private Kit access to your location.",
-    "import":"Import",
-    "export":"Export",
-    "news":"news",
-    "latest_news":"Latest News",
-    "url_info":"For more information visit the Private Kit hompage:",
+    "start_logging":"INICIAR",
+    "stop_logging":"PARAR",
+    "logging_message":"É registrado sua localização atual a cada cinco minutos. Suas informações de localização NÃO sairão do seu telefone.",
```
Comment (remintz) [inline app/locales/pt/locationTracking.json]: Suggestion for translation: "Sua localização atual está sendo registrada a cada cinco minutos. ..."

File: app/locales/pt/locationTracking.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+    "not_logging_message":"NOTA: Depois de clicar neste botão, você pode ser solicitado para liberar o acesso da sua localização ao Private Kit.",
+    "import":"Improtar",
```
Comment (remintz) [inline app/locales/pt/locationTracking.json]: typo: Importar

File: app/locales/pt/locationTracking.json (hunk 1 above)
Comment (douglara) [reply app/locales/pt/locationTracking.json]: Thanks friend applied.

File: app/locales/pt/locationTracking.json (hunk 2 above)
Comment (douglara) [reply app/locales/pt/locationTracking.json]: Thanks! fixed =)

**Comment by penrods** [comment]
@remintz, Are you a native speaker?  Leave a comment if you approve of the translation -- if so I'll go ahead and merge!

**Comment by remintz** [comment]
> @remintz, Are you a native speaker? Leave a comment if you approve of the translation -- if so I'll go ahead and merge!

Yes I am.

**Comment by remintz** [comment]
> @remintz, Are you a native speaker? Leave a comment if you approve of the translation -- if so I'll go ahead and merge!

And yes... I approve

**Comment by carlosbueno360** [comment]
I am a native speaker to, and I approve it too

**Comment by RenanErnest** [comment]
I am also a native speaker and I Approve it.

**Comment by tremblerz** [comment]
great work guys

**Pull request merged**